httpd/handler.go

```diff
@@ -291,10 +281,7 @@ type dataNodeJSON struct {
 }
 
 func isAuthorizationError(err error) bool {
```

Comment:
This function was actually never returning true, regardless.  This fixes the issue.